.github/workflows/community-releases.yml

```diff
@@ -0,0 +1,98 @@
+name: Docker Community Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to tag docker images'
+        required: true
+        type: string
```

Comment:
I think that it would be nice to have an option to build and release just specified images. I can imagine that when some images fail we would need to have an option just to build for example two `novu/api, novu/worker` after we have applied the fix. WDYT?